exampleCourse/questions/demo/randomFakeData/server.py

```diff
@@ -15,7 +15,7 @@ def generate(data):
     employees = []
 
     for alias, birthday in zip(
-        ["youngest", "employee1", "employee2", "employee3"], birthdays
+        ["youngest", "employee1", "employee2", "employee3"], birthdays, strict=False
```

Comment:
are we sure we want `strict=False` here and not `strict=True`?

Reply:
Resolved in b6cfaab71